implementations/php-json-schema/bowtieJsonSchema.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+use Opis\JsonSchema\{
+    Validator
+};
+
+$STARTED = false;
+
+$validator = new Validator();
+$resolver = $validator->loader()->resolver();
+
+function start($request)
+{
+    if ($request->version !== 1) {
+        throw new Exception('Wrong version!');
+    }
+
+    $GLOBALS['STARTED'] = true;
+
+    $response = [
+        'ready' => true,
+        'version' => 1,
+        'implementation' => [
+            'language' => 'php',
+            'name' => 'json-schema',
+            'homepage' => 'https://opis.io/json-schema',
+            'issues' => 'https://github.com/opis/json-schema/issues',
+            'dialects' => [
+                'http://json-schema.org/draft/2020-12/schema#',
```

Comment:
These are the wrong dialect URIs (or well, they're not precisely the same ones as the correct URIs), which is likely why you're seeing `UnknownDialect` errors. The correct ones do not have hashes, see e.g. https://github.com/bowtie-json-schema/bowtie/blob/main/implementations/python-jsonschema/bowtie_jsonschema.py#L47-L48 -- and yeah good to trust the error message always, if it's saying "this is wrong" double check carefully whether it's possible it's correct.